08-rails/README.md

```diff
@@ -10,17 +10,21 @@
     - [Rails Templates](rails-template-setup.md)
     - [Configuring Rails to use Postgres](how-to-use-postgres.md)
     - [Live Code Intro](live-code-intro.md)
-1. [Controllers](routes-controllers.md)
+1. [Controllers and the Index Action](controllers.md)
 1. [ERB & Layouts](ERB-and-Layouts.md)
-1. [Route Parameters](route-parameters.md)
+1. [Intro to Testing Controllers: Adding Tests to the Index Action](intro-to-testing-controllers.md)
+1. [Route Parameters & the Show Action](route-parameters.md)
+1. [Intro to the Show Action and its Tests](show-action-intro.md)
 1. [Requests and Responses](requests-and-responses.md)
+1. [What is a Database?](database-intro.md)
 1. [Active Record Migrations](active-record-migrations.md)
 1. [Active Record Models](active-record-models.md)
     - [Exercise: Hike Finder Single Model](exercises/hike-finder/single-model.md)
```

Comment:
The intro to databases link is broken

Reply:
I... I don't know how to resolve this? I am fairly certain that this link is working; on this PR is moving the file so that this link works, and it's working on GitHub's file preview. What are you observing?